android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -1044,7 +1047,54 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
         result.success(HMSTrackExtension.toDictionary(peer?.getTrackById(trackId!!)))
     }
 
+    private fun setPlaybackAllowedForTrack(call: MethodCall, result: Result){
+        val trackId : String? = call.argument<String>("track_id")
+        val isPlaybackAllowed : Boolean = call.argument<String>("is_playback_allowed") as Boolean
+        val trackKind : String? = call.argument<String>("track_kind")
+
+        val peersList = hmssdk!!.getRemotePeers()
```

Comment:
`getAllAudioTracks` & `getAllVideoTracks` util functions are provided from SDKs. Use that instead of this